#! /bin/sh -e
## 20_Makefile_lib64.dpatch by Craig Small <csmall@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Removed entry about /lib64 as it breaks the sparc

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argum
ent"
                exit 1;;
esac

exit 0
@DPATCH@
--- procps/Makefile.orig	Thu Apr  1 22:07:46 2004
+++ procps/Makefile	Thu Apr  1 22:08:31 2004
@@ -31,7 +31,7 @@
 install  := install -D --owner 0 --group 0
 
 # Lame x86-64 /lib64 and /usr/lib64 abomination:
-lib64    := lib$(shell [ -d /lib64 ] && echo 64)
+lib64    := lib
 
 usr/bin                  := $(DESTDIR)/usr/bin/
 bin                      := $(DESTDIR)/bin/
@@ -110,7 +110,7 @@
 ALL_CFLAGS += $(call check_gcc,-Wpadded,)
 
 # Be 64-bit if at all possible.
-ALL_CFLAGS += $(call check_gcc,-m64,)
+#ALL_CFLAGS += $(call check_gcc,-m64,)
 
 endif
 endif
